docs/concepts/native-interop.rst

```diff
@@ -0,0 +1,260 @@
+Native interoperability
+=======================
+
+In this document we will dive a little bit deeper into all three ways of doing 
+"native interoperability" that are available on the .NET platform. 
+
+To refresh your memory, the two main ways of doing native interoperability are:
+
+* Platform Invoke (P/Invoke)
+* COM interoperability
```

Comment:
If this is a general-purpose .NET Primer, which I understood it is, then yes, COM interop should be there, because it is still a thing in the full .NET Framework. If this is a primer for the Core, then we should remove it.